### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by naikrovek** [wezterm/wezterm#650] [General]
I was looking at this expecting it to be a mostly featureless terminal and I was expecting to not find what I was looking for.

I was wrong.

Thank you for this, thank you for releasing it openly, and ... yeah.

Not an issue, obviously, just a thank you.

**Comment by naikrovek** [comment]
Just wanted to reiterate my "thank you."  I've been sponsoring this project for _ages_ and I'm going to continue to do so.  I love this terminal and I am thankful for it.  tagging @wez so he gets notified and feels the love I have for this tool.

**Reply by wez** [reply]
Thank you!
